$OpenBSD: patch-myocamlbuild_ml,v 1.1.1.1 2014/10/25 17:05:14 chrisz Exp $

applications/distcheck.ml depends on StdOptions,
whose .cmi is only found in doseparse.

--- myocamlbuild.ml.orig	Tue Sep  2 23:48:11 2014
+++ myocamlbuild.ml	Tue Sep  2 23:52:55 2014
@@ -14,10 +14,10 @@ let libraries = [
   "boilerplate"; "boilerplateNoRpm";
 ];;
 
-let doselibs = "doselibs" ;;
+let doselibs = ["doselibs"; "doseparse"] ;;
 
 (* everybody can see the library dir *)
-List.iter (fun d -> Pathname.define_context d [doselibs]) modules_dirs ;;
+List.iter (fun d -> Pathname.define_context d doselibs) modules_dirs ;;
 
 let _ = dispatch begin function
    | After_rules ->
